src/_TextInputInnerVisualSlot.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {Box, Spinner} from '.'
+import {TextInputNonPassthroughProps} from './TextInput'
+
+const TextInputInnerVisualSlot: React.FC<{
```

Comment:
is it more like `TextInputReplaceLoader` or something like that.

Reply:
I still feel like it's more about the visual than the loader.